cypress/integration/Signup/neetoAuthSignup.spec.js

```diff
@@ -0,0 +1,107 @@
+ /// <reference types="cypress" />
+
+import { sigupSelectors,  infoDetailsSelectors, OTPSelectors, passwordSelector, organisationSelector, cancelSelectors, profileDashboard, messageSelector } from '../../constants/selectors/selectors'
+import { texts } from '../../constants/texts/texts'
+import { routes } from '../../constants/routes/routes'
+import { formFill } from '../../utils/formFill';
+const faker = require('faker');
+
+describe("NeetoAuth Signup Test Suite",() => {
+  let orgName = "BigBinary";
+  let invalidOTP = "123756";
+  let invalidPassword = 'meow';
+  let randomPassword = faker.internet.password();
+  let randomOrgName = faker.random.word();
+
+  beforeEach(() => {
+     formFill();
+  });
+
+  it("Should be able to signup",() => {
```

Comment:
The description of test suite and all test cases should be in lower case.
```suggestion
  it("should be able to signup",() => {
```